src/docs/arch-eviction.dox

```diff
@@ -33,6 +33,13 @@ the eviction server walks the tree to find pages to evict and evict these pages
 cache. Pages that are being marked for forced eviction go onto the urgent
 queue, and these pages take precedence over the pages from the ordinary eviction queues.
 
+Pages will go into the urgent queue only if the following conditions are met; otherwise,
+the page can be evicted by the application thread itself:
+- The operation has disabled eviction or splitting.
+- The session is preventing from reconciling.
+
+Otherwise, it will attempt to release and evict it.
```

Comment:
I didn't quite get the `release` part here. I feel it is better to remove and just mention `attempt to evict it` or provide more details of what is `release`.

Reply:
thank you for the suggestion - I've clarified the text and removed the ambiguous "release"